addons/cookoff/functions/fnc_cookOffEffect.sqf

```diff
@@ -183,5 +184,13 @@ if (isServer) then {
             [2 + random 1], 1, 0, "", "", _obj
         ];
     };
-}, 0, [_obj, _jet, _ring, _time, CBA_missionTime, _light, _fireSelection, _sound]] call cba_fnc_addPerFrameHandler;
+    
+    private _tiPars = getVehicleTIPars _obj; 
+    _obj setVehicleTIPars [
+        ((_tiPars select 0) + (_intensity * 0.01))/1.005, 
+        ((_tiPars select 1) + (_intensity * 0.004))/1.002, 
+        ((_tiPars select 2) + (_intensity * 0.01))/1.005
+    ];
```

Comment:
`vectorAdd` will be faster than doing `select` three times.
```suggestion
    _obj setVehicleTIPars (_tiPars vectorAdd [
        (_intensity * 0.01) / 1.005, 
        (_intensity * 0.004) / 1.002, 
        (_intensity * 0.01) / 1.005
    ]);
```

Reply:
math is different
`(current + add) / 1.05` 
vs
`current + (add/ 1.05)`